index.html

```diff
@@ -9,25 +9,55 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
-<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
     <title>Document</title>
   </head>
   <body>
-    <h1 class="display-6">Awesome books</h1>
-    <ul id="book-list"></ul>
-    <hr class="add-book-hr">
-    <h2>Add a new book</h2>
-    <ul id="add-list">
-      <li>
-        <input type="text" name="title" id="title" placeholder="Title" />
-      </li>
-      <li>
-        <input type="text" name="author" id="author" placeholder="Author" />
-      </li>
-      <li>
-        <button id="add-button" type="button">Add</button>
-      </li>
-    </ul>
+    <header>
+      <ul id="menu-list">
+        <li>Awesome Books</li>
+        <li>
+          <a class="list-btn first-two">List</a>
+        </li>
+        <li>
+          <a class="add-new-btn first-two">Add new</a>
+        </li>
+        <li>
+          <a class="contact-btn">Contact</a>
+        </li>
+      </ul>
+      <p class="date-and-time"></p>
+    </header>
+    <section id="book-list-section" >
+      <h1 class="display-6">Awesome books</h1>
+      <ul id="book-list"></ul>
+    </section>
+    <section id="add-book-section" class="hidden">
+      <h2>Add a new book</h2>
+      <ul id="add-list">
+        <li>
+          <input type="text" name="title" id="title" placeholder="Title" />
+        </li>
+        <li>
+          <input type="text" name="author" id="author" placeholder="Author" />
+        </li>
+        <li>
+          <button id="add-button" type="button">Add</button>
+        </li>
+      </ul>
+    </section>
+    <section id="contact-info-section" class="hidden">
+      <h2>Contact information</h2>
+      <p>Do have any questions or you just want to say "Hello"? You can reach out to us!</p>
+      <ul>
+        <li>Our email: <a href="mailto:mail@mail.com">mail@mail.com</a></li>
+        <li>Our phone number: 004398234097</li>
+        <li>Our address: Streetname 22, 84503, City, Country</li>
+      </ul>
+    </section>
+    <footer>
+      <p>Copyright &copy; ...</p>
+    </footer>
```

Comment:
Please fix the position of the footer at the bottom of the page as it can be seen in the wireframe.
Yours:
![footer](https://user-images.githubusercontent.com/35504149/143415789-fce525bb-c29e-4a89-bb29-ec8cab46f577.png)

Reply:
can I fix it using bootstrap?